packages/x-charts/src/internals/plugins/featurePlugins/useChartKeyboardNavigation/useChartKeyboardNavigation.selectors.ts

```diff
@@ -39,6 +41,21 @@ export const selectorChartsIsKeyboardNavigationEnabled = createSelector(
   (keyboardNavigationState) => !!keyboardNavigationState?.enableKeyboardNavigation,
 );
 
+export const selectorChartsIsFocused = createSelectorX(
+  selectKeyboardNavigation,
+  (keyboardNavigationState, item: FocusedItemData) => {
+    const focusedItem = keyboardNavigationState?.item;
+
+    return Boolean(
+      item &&
+        focusedItem &&
+        focusedItem.type === item.seriesType &&
```

Comment:
Inconsistencey between `SeriesItemIdentifier` and `FocusedItemData`, maybe you should use the same type for both.